components/formats-gpl/src/loci/formats/in/GatanReader.java

```diff
@@ -378,7 +378,7 @@ else if (n == 3) {
             }
             else {
               if (dataType == 10) in.skipBytes(length);
-              else value = DataTools.stripString(in.readString(length * 2));
+              else value = in.readString(length * 2);
```

Comment:
Sorry, maybe I don't follow - if this reader has a readString(int) method that mostly works, could it not be used here instead of changing the RandomAccessInputStream.readString implementation?